css/navbar.css

```diff
@@ -9,9 +9,50 @@ header > nav:first-child {
     background-position: 80% 0%;
 }
 
-#CollapsingNavbar > div:first-child {
-    display: flex;
-    justify-content: flex-end;
+
+@media (min-width: 993px) {
```

Comment:
Just a question, never seen selectors with a single pixel difference between max-width and min-width. Is that normal or a typo?

Reply:
@DiegoPino This is normal in my experience (difference of a pixel when using min-width vs. max-width). Min-width being the best practice, mobile-first approach, but sometimes using max-width in a query is necessary.

Ideally you could use SASS mixins for Bootstrap media queries like `@include media-breakpoint-up(sm) { ... }` which deals with that pixel overlap and is less error prone but this assignment was just CSS.

See https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints